src/main/java/com/shopping_app/shoppingApp/Exceptions/BaseException.java

```diff
@@ -0,0 +1,17 @@
+package com.shopping_app.shoppingApp.Exceptions;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public class BaseException extends RuntimeException {
+    private HttpStatus httpStatus;
```

Comment:
You can leave a blank line after this to make it look more readable

Reply:
Sure